s3-launch-transfers.sh

```diff
@@ -302,7 +302,7 @@ function curl_s3 {
         echo "RETRY attempt (${retry_count})" `date` | tee -a $OPEN
     fi
     curl_cmd=(
-	$CURL -vv "${copts[@]}"
+	$CURL -v "${copts[@]}"
```

Comment:
This was bothering me too, so in my local clone for the EoT project I had verbosity removed completely. Please make sure that reducing the level from `-vv` to `-v` indeed gets rid of the annoying spam of warnings.

Reply:
I tested it on a number of machines and `-v` was all I needed, but when Will tested it, it did not work on some. I think the curl version may effect it. 

I would be happy to get rid of all verbosity if you think that would be a good idea.

Reply:
> I would be happy to get rid of all verbosity if you think that would be a good idea.

One possibility is to add an option to enable or disable it, but I am okay getting rid of it really.